scripts/setup_colab.sh

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env bash
+# This script installs garage on https://colab.research.google.com
+#
+# NOTICE: To keep consistency across this script, scripts/setup_macos.sh and
+# docker/Dockerfile.base, if there's any changes applied to this file,
+# specially regarding the installation of dependencies, apply those same
+# changes to the mentioned files.
+
+# Exit if any error occurs
+set -e
+
+### START OF CODE GENERATED BY Argbash v2.6.1 one line above ###
+die()
+{
+  local _ret=$2
+  test -n "$_ret" || _ret=1
+  test "$_PRINT_HELP" = yes && print_help >&2
+  echo "$1" >&2
+  exit ${_ret}
+}
+
+begins_with_short_option()
+{
+  local first_option all_short_options
+  all_short_options='h'
+  first_option="${1:0:1}"
+  test "$all_short_options" = "${all_short_options/$first_option/}" && \
+    return 1 || return 0
+}
+
+
+
+# THE DEFAULTS INITIALIZATION - POSITIONALS
+_positionals=()
+# THE DEFAULTS INITIALIZATION - OPTIONALS
+_arg_mjkey=
+_arg_modify_bashrc="off"
+_arg_gpu="off"
+
+print_help ()
+{
+  printf '%s\n' "Installer of garage for Linux."
+  printf 'Usage: %s [--mjkey <arg>] [--(no-)modify-bashrc] ' "$0"
+  printf '[--(no-)gpu] [-h|--help]\n'
+  printf '\t%s\n' "--mjkey: Path of the MuJoCo key (no default)"
+  printf '\t%s' "--gpu,--no-gpu: Install GPU support of Tensorflow "
+  printf '%s\n' "(off by default)"
+  printf '\t%s' "--modify-bashrc,--no-modify-bashrc: Set environment variables "
+  printf '%s\n' "required by garage in .bashrc (off by default)"
+  printf '\t%s\n' "-h,--help: Prints help"
+}
+
+parse_commandline ()
+{
+  while test $# -gt 0
+  do
+    _key="$1"
+    case "$_key" in
+      --mjkey)
+        test $# -lt 2 && \
+          die "Missing value for the optional argument '$_key'." 1
+        _arg_mjkey="$2"
+        shift
+        ;;
+      --mjkey=*)
+        _arg_mjkey="${_key##--mjkey=}"
+        ;;
+      --no-modify-bashrc|--modify-bashrc)
```

Comment:
Besides mjkey and h, all the other arguments are not used here, so it's better to remove all the code associated to them. The code for the arguments in this script was generated with argbash, so I would recommend generating them again with only the mjkey and h arguments.

Reply:
Ups, it seems `gpu` can be used in colab (the future is now):
https://colab.research.google.com/notebooks/gpu.ipynb
So, also make sure to include the `gpu` argument please.